_posts/2016-11-01-inifile.md

```diff
@@ -167,6 +167,47 @@ Goals are defined with close polygons, with  *the last vertex is equal to the fi
 
 - `file` file containing further goals. See [goals.xml](2019-04-12-goals.html)
 
+### Waiting Area
+
+Addional goals which also can be defined **inside** the geometry.
+When the waiting area is reached, pedestrians may wait till they move to one of the specified next goal (decided individually for each ped).
+Waiting either depends on time or till a specific door opens.
+They should _NOT_ overlap with any walls or be inside rooms.
+
+Waiting areas are defined with close polygons, with  *the last vertex is equal to the first one*.
+
+```xml
+ <routing>
+    <goals>
+			<waiting_area caption="wa1" id="1" waiting_time="20" min_peds="5" max_peds="10" is_open="true" room_id="0"  subroom_id="1" global_timer="false" transition_id="1">
+				<polygon>
+					<vertex px="11" py="1" />
+					<vertex px="14" py="1" />
+					<vertex px="14" py="4" />
+					<vertex px="11" py="4" />
+					<vertex px="11" py="1" />
+				</polygon>
+				<next_wa id="2" p="0.75"/>
+				<next_wa id="3" p="0.25"/>
+			</waiting_area>
+        <file>goals.xml</file>
+    </goals>
+ </routing>
+```
+
+- `file`: file containing further goals/waiting areas. See [goals.xml](2019-04-12-goals.html)
+- `waiting_time`: the time pedestrians wait inside the waiting area
+- `min_peds`: the number of pedestrians needed in the waiting area to start the timer (if `global_timer` = false)
+- `max_peds`: the max number of pedestrians allowed inside the waiting area. **Important:** to avoid unbehaviour `max_peds` should not exceed the number of pedestrians heading for the particulat waiting area. Hence `max_peds(WA1)` <= `max_peds(WA2)`.
```

Comment:
- unbehavior --> undefined behavior
- the particulat waiting area --> another waiting area?